ibm/service/kubernetes/resource_ibm_container_vpc_worker_pool.go

```diff
@@ -232,6 +240,33 @@ func ResourceIBMContainerVPCWorkerPoolValidator() *validate.ResourceValidator {
 
 func resourceIBMContainerVpcWorkerPoolCreate(d *schema.ResourceData, meta interface{}) error {
 
+	if ioc, ok := d.GetOk("import_on_create"); ok && ioc.(bool) {
+		if cluster, ok := d.GetOk("cluster"); ok {
+			clusterID := cluster.(string)
```

Comment:
the cluster is a required parameter, no need to use GetOk. you can reuse the code from below
```suggestion
	clusterNameorID := d.Get("cluster").(string)
	if ioc, ok := d.GetOk("import_on_create"); ok && ioc.(bool) {
```